example/libs/subdir/lib-c/src/lib_c/lib_b/core_test.clj

```diff
@@ -0,0 +1,9 @@
+(ns lib-c.core-test
```

Comment:
This file isn't in the right place - there's both an extra `lib_b` path segment, and it should be under `test` not `src`. That said, exercising the tests here isn't important so I'd prefer to just drop it.